Can't build

```
   Compiling pnet_packet v0.25.0
error[E0425]: cannot find function `register` in crate `pnet_macros`
  --> /home/mkpankov/.cargo/registry/src/github.com-1ecc6299db9ec823/pnet_packet-0.25.0/build.rs:27:30
   |
27 |                 pnet_macros::register(&mut registry);
   |                              ^^^^^^^^ not found in `pnet_macros`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `pnet_packet` due to previous error
➜  ja3-rs git:(master) cargo build --release --locked
   Compiling pnet_packet v0.25.0
error[E0425]: cannot find function `register` in crate `pnet_macros`
  --> /home/mkpankov/.cargo/registry/src/github.com-1ecc6299db9ec823/pnet_packet-0.25.0/build.rs:27:30
   |
27 |                 pnet_macros::register(&mut registry);
   |                              ^^^^^^^^ not found in `pnet_macros`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `pnet_packet` due to previous error
```
```
➜  ja3-rs git:(master) rustc --version
rustc 1.55.0 (c8dfcfe04 2021-09-06)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,7 +24,7 @@ log = "0.4.8"
 md5 = "0.7.0"
 pcap = { version = "0.7.0", optional = true }
 pcap-parser = "0.9.2"
-pnet = "0.25.0"
+pnet = "0.28.0"
 tls-parser = "0.9.2"
 
 [dev-dependencies]

